### PR TITLE
batch metadata update

### PR DESCRIPTION
**Pull request by mhagel**
## Description

Closes: regen-network/regen-registry#870

- removes verification and monitoring docs from Credit Batch metadata
- adds Additional Certifications
- cleans up some lint errors

Testable redwood ledger metadata at https://deploy-preview-910--regen-registry.netlify.app/credit-batches/C01-20190101-20210101-008

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**.*

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles 
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by mhagel** [comment]
Testable redwood ledger metadata at https://deploy-preview-910--regen-registry.netlify.app/credit-batches/C01-20190101-20210101-008

File: web-registry/src/components/molecules/BatchMetadata.tsx [hunk 1]
```diff
@@ -10,34 +10,36 @@ export const BatchMetadata: React.FC<{
   data?: BatchMetadataLD;
   sx?: SxProps<Theme>;
 }> = ({ sx, data }) => {
-  const monitoringReport = data?.['regen:batchMonitoringReport'];
-  const verificationReport = data?.['regen:batchVerificationReport'];
+  const additionalCertifications =
+    data?.['regen:additionalCertifications']?.['@list'];
```
Comment (blushi) [inline web-registry/src/components/molecules/BatchMetadata.tsx:14]: I believe we might not actually need the @list, see https://github.com/regen-network/regen-registry-standards/pull/15

File: web-registry/src/types/rdf/index.ts [hunk 2]
```diff
@@ -1,9 +1,14 @@
 export interface URL {
-  '@type': 'http://schema.org/URL';
+  '@type': string;
```
Comment (blushi) [inline web-registry/src/types/rdf/index.ts:2]: probably best to restrict it to:
```suggestion
  '@type': 'http://schema.org/URL' | 'schema:URL';
```

**Review by blushi — CHANGES_REQUESTED** [review]

File: web-registry/src/types/rdf/index.ts (hunk 2 above)
Comment (mhagel) [reply web-registry/src/types/rdf/index.ts:2]: Agreed

File: web-registry/src/components/molecules/BatchMetadata.tsx (hunk 1 above)
Comment (mhagel) [reply web-registry/src/components/molecules/BatchMetadata.tsx]: Ok, I understand how this works now. Fixed.

**Review by blushi — APPROVED** [review]

**Review by victordelval — APPROVED** [review]

**Comment by mhagel** [comment]
@mergifyio backport redwood hambach

**Pull request merged**